### PR TITLE
FixTypo

### PR DESCRIPTION
**Pull request by dota17**
FixTypo:
occured -> "occurred"
transfered -> "transferred"
recieved -> "received"
prefered -> "preferred"
transfered -> "transferred"
occuring -> "occurring"
cant -> "can not","can't","cannot"

### CODE REVIEW

File: src/main/java/org/java_websocket/util/Base64.java [hunk 1]
```diff
@@ -487,17 +487,17 @@ private Base64(){}
...
-     * The actual number of significant bytes in your array is
+     * The actual number of significan't bytes in your array is
      * given by <var>numSigBytes</var>.</p>
 	 * <p>This is the lowest level of the encoding methods with
 	 * all possible parameters.</p>
      *
      * @param source the array to convert
      * @param srcOffset the index where conversion begins
-     * @param numSigBytes the number of significant bytes in your array
+     * @param numSigBytes the number of significan't bytes in your array
```
Comment (PhilipRoman) [inline src/main/java/org/java_websocket/util/Base64.java:500]: False match

Comment (PhilipRoman) [inline src/main/java/org/java_websocket/util/Base64.java:500]: Incorrect match

File: autobahn reports/clients/index.html [hunk 2]
```diff
@@ -2263,7 +2263,7 @@ <h2>Case 6.4.1</h2>
 Note that PART1 and PART3 are valid UTF-8 in themselves, PART2 is a 0x11000 encoded as in the UTF-8 integer encoding scheme, but the codepoint is invalid (out of range).
 <br><br>MESSAGE PARTS:<br>
 PART1 = κόσμε (cebae1bdb9cf83cebcceb5)<br>
-PART2 = � (f4908080)<br>
+PART2 = ���� (f4908080)<br>
```
Comment (PhilipRoman) [inline autobahn reports/clients/index.html:2266]: Not sure what happened here

Comment (marci4) [reply autobahn reports/clients/index.html:2266]: Pretty sure it is the generator and will break when next updated

File: src/test/java/org/java_websocket/framing/CloseFrameTest.java [hunk 3]
```diff
@@ -42,7 +42,7 @@ public void testConstructor() {
         CloseFrame frame = new CloseFrame();
         assertEquals("Opcode must be equal", Opcode.CLOSING, frame.getOpcode());
         assertEquals("Fin must be set", true, frame.isFin());
-        assertEquals("TransferedMask must not be set", false, frame.getTransfereMasked());
+        assertEquals("transferredMask must not be set", false, frame.getTransfereMasked());
```
Comment (dota17) [inline src/test/java/org/java_websocket/framing/CloseFrameTest.java:45]: Is it ok to fix? The field name in frame `FramedataImpl1` was `transferemasked`.

File: autobahn reports/servers/index.html [hunk 4]
```diff
@@ -4112,7 +4112,7 @@ <h2>Case 7.1.5</h2>
       <h2>Case 7.1.6</h2>
       <a class="up" href="#top">Up</a>
       <p class="case_text_block case_desc"><b>Case Description</b><br/><br/>Send 256K message followed by close then a ping</p>
-      <p class="case_text_block case_expect"><b>Case Expectation</b><br/><br/>Case outcome depends on implementation defined close behavior. Message and close frame are sent back to back. If the close frame is processed before the text message write is complete (as can happen in asynchronous processing models) the close frame is processed first and the text message may not be received or may only be partially recieved.</p>
```
Comment (PhilipRoman) [inline autobahn reports/servers/index.html:4115]: Since this is a generated report, it probably shouldn't be edited manually.

**Pull request merged**